pvlib/test/test_ivtools.py

```diff
@@ -102,6 +109,29 @@ def test_fit_sdm_cec_sam(get_cec_params_cansol_cs5p_220p):
                 cells_in_series=1, temp_ref=25)
 
 
+@requires_scipy
+def test_fit_sdm_desoto(get_test_specs_params):
+    result = ivtools.fit_sdm_desoto(cells_in_series=60,
```

Comment:
This is where you might test "all" the `root` solvers:
```python
for solver_method in (‘hybr’, ‘lm’, ‘broyden1’, ‘broyden2’,‘anderson’,‘linearmixing’, ‘diagbroyden’, ‘excitingmixing’, ‘krylov’, ‘df-sane’):
```
However, this is somewhat fragile w.r.t. scipy updating the various solvers.

Reply:
We could restrict the `method` values, if we decide we're not testing/supporting some of the options.

Reply:
Good point. 
I did not explore that before, but I realize that only two of the methods proposed in scipy.optimize.root() actually work for fit_sdm_desoto, namely 'hybr' and 'lm'.

If we don't mention the solver_method of scipy.optimize.root(), it may be less fragile maybe ? I propose to remove the kwarg solver_method and other later reference of it. This way, scipy.optimize.root() will just choose the default value (currently 'hybr'), and if scipy is updated, we can expect scipy to still use a robust method as default value.

And moreover it would simplify a bit the code of fit_sdm_desoto. I don't think that being able to choose the rooting method is a very important feature for most users of pvlib. What do you think ?

Reply:
I would be ok with using the default solver.

Reply:
> I would be ok with using the default solver.

I am also ok with this now; thanks for this discussion.

> Probably should mention this in the docstring.

yes.